Issue-37 - Added a simple type='colorbox' display field

DougSchmidt-AI opened the pull request:
This just shows a 80px x 20px rectangle bound to the data value, which can be any support HTML color spec (named color, hex string, rrgb(), or hsl() expression.

dsternlicht commented:
I'm not sure what's the use case here, but @DougSchmidt-AI could you please add it to the documentation as well?

DougSchmidt-AI commented:
OK, I reworked this a bit and updated the docs.

Now the display field `colorbox` type can be used to show both the hex color and the text value, like this:

![image](https://user-images.githubusercontent.com/54773/52141530-e3823300-260a-11e9-92ef-c4d7f736f9ec.png)

And added support for `<input type="color">` fields, to use the platforms color selector to pick and #RRGGBB hex string.

![image](https://user-images.githubusercontent.com/54773/52141599-15939500-260b-11e9-8410-632ddad237be.png)
